include/kframework/state-utils.md

```diff
@@ -226,6 +253,9 @@ The `"rlp"` key loads the block information.
          <baseFee>           _ => #asWord(#parseByteStackRaw(HF)) </baseFee>
          <ommerBlockHeaders> _ => BU                              </ommerBlockHeaders>
 
+    rule <k> load "genesisRLP": [ [ HP, HO, HC, HR, HT, HE:String, HB, HD, HI, HL, HG, HS, HX, HM, HN, HF, WR, .JSONs ], _, _, _, .JSONs ] => .K ... </k>
+         <blockhashes> .List => ListItem(#blockHeaderHash(HP, HO, HC, HR, HT, HE, HB, HD, HI, HL, HG, HS, HX, HM, HN, HF, WR)) ListItem(#asWord(#parseByteStackRaw(HP))) ... </blockhashes>
+
```

Comment:
Similarly, could we reduce to the old rule here and have less code duplication? Maybe it makes less sense here since there is less in common anyway.